ISPN-12258 documentation updates (#548)
infinispan/infinispan-operator: documentation/asciidoc/topics/proc_configuring_sites.adoc

INFO  [org.infinispan.XSITE] (jgroups-7,{example_crd_name}-0-<id>) ISPN000439: Received new x-site view: [NYC, LON]
----

.Next steps

> If your clusters have formed an x-site view, you can start adding backup

Reviewer: ```suggestion
If your clusters have formed a cross-site view, you can start adding backup
```
